ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcClientProtocolService.java

```diff
@@ -48,8 +48,8 @@
 import java.util.function.Consumer;
 import java.util.function.Supplier;
 
-class GrpcClientProtocolService extends RaftClientProtocolServiceImplBase {
-  private static final Logger LOG = LoggerFactory.getLogger(GrpcClientProtocolService.class);
+public class GrpcClientProtocolService extends RaftClientProtocolServiceImplBase {
+  public static final Logger LOG = LoggerFactory.getLogger(GrpcClientProtocolService.class);
```

Comment:
@captainzmc , sorry that GrpcClientProtocolService is not a public API.  We should not change it for an Ozone unit test.  We should fix the test instead.